### PR TITLE
Add slightly friendlier messages

### PR DESCRIPTION
**Pull request by mrchrisadams**
- Tell users where to go next after a model has run to get their data.
- Give clearer direction on the description
- make login errors sound more like our issue, not their fault.

### CODE REVIEW

File: src/cljs/witan/ui/strings.cljs [hunk 1]
```diff
@@ -37,7 +37,7 @@
    :thanks                         "Thanks"
    :signing-in                     "Signing in..."
    :sign-in-failure                "There was a problem with your details. Please try again."
-   :api-failure                    "There was a problem with the service. Please try again. If the problem persists, please contact us." ;; TODO add link?
+   :api-failure                    "Sorry, we're having a problem with the service. Please try again. If the problem persists, please contact us. at witan@mastodonc.com" ;; TODO add link?
```
Comment (acron0) [inline src/cljs/witan/ui/strings.cljs]: Remove the full-stop after 'please contact us'

File: src/cljs/witan/ui/strings.cljs [hunk 2]
```diff
@@ -51,8 +51,8 @@
    :new                            "New"
    :upload                         "Upload"
    :upload-new-data                "Upload new data"
-   :forecast-changes-text          "There are changes to this projection. If you're ready, you can now run this projection - the model will be processed with your selected inputs and new outputs will be generated."
-   :forecast-in-progress-text      "This version is currently being generated. During this time you will be unable to make changes or download results. This can take several minutes."
+   :forecast-changes-text          "Okay, we've recorded your changes. When you're ready, hit 'Run this projection' to generate your new outputs to download."
```
Comment (acron0) [inline src/cljs/witan/ui/strings.cljs]: 'for download' ?

File: src/cljs/witan/ui/strings.cljs [hunk 3]
```diff
@@ -71,8 +71,8 @@
    :browser-upload-error           "An error occurred whilst trying to upload the file. Please try again and if this problem persists, contact us." ;; TODO wants email
    :downloads                      "Downloads"
    :download                       "Download"
-   :new-version-no-downloads       "Downloads are not yet available. After configuring and running the projection for the first time your downloads will appear in this section."
-   :in-progress-no-downloads       "Downloads are not yet available because the model is still running. This can take several minutes so please check again shortly."
+   :new-version-no-downloads       "Once you have run your first projection, your data will be available here. Start this at 'Inputs' section"
```
Comment (acron0) [inline src/cljs/witan/ui/strings.cljs]: at the\* 'Inputs Section'

**Comment by mrchrisadams** [comment]
Thanks for checking. Have updated the typos accordingly.

**Comment by acron0** [comment]
:shipit:

**Pull request merged**